### ISSUE TITLE
Error report fails to Send

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#4154]
### Bug description
Many times now I have encountered failure to send an Audacity error report, the ET-phone-home dialog, when I click the **Send** button.

This has worked for me in the past - it is only in recent times that this seems to be happening.  A couple of posters on the Forum have also reported similar recently.

![image](https://user-images.githubusercontent.com/50205984/211068529-86a4f471-c203-4ad9-9302-3f2f8a4b23a4.png)

### Steps to reproduce
1.  do something that creates and error report dialog
2. Click Send
3. Observe: failure message

### Expected behavior
Error log to be sent to base

### Actual behavior
Error log not sent to base

### Audacity Version
3.2.3 and 3.3.0 beta (and @Paul-Licameli 's latest Parallel processing branch)
### Operating system
Windows 10

### Additional context
None

**Comment by crsib** [comment]
The issue has been reported to our SA team.

**Comment by LWinterberg** [comment]
Does this still happen?

**Comment by crsib** [comment]
We still get messages about it

**Comment by petersampsonaudacity** [comment]
Yes it does - I still get it and we get Forum reports of this failure to send

**Comment by petersampsonaudacity** [comment]
Seems to be working here on my W11 Zürich PC

**Comment by petersampsonaudacity** [comment]
Seems to be working these days on my Manchester W10 PC

**Comment by LWinterberg** [comment]
I guess this is done then

**Comment by petersampsonaudacity** [comment]
@crsib @LWinterberg @chinakov 

It's started happening again - consistently when I test the crash on record immediately after launch on the latest alphas:
 **Windows: Record crashes latest 3.4.0 alphas #4996** 

![image](https://github.com/audacity/audacity/assets/50205984/051fb003-65ed-419c-b9cb-9921d03772d0)

**Comment by dozzzzer** [comment]
Confirm the issue is present again on Windows 11.

**Comment by LWinterberg** [comment]
I'm getting this right now. Interestingly, posting the stack trace into github doesn't work either because it runs over the character limit. Is sentry perhaps constrained to 2^16 characters for crash reports?

**Comment by JohnColket** [comment]
Also just reported on forum here: https://forum.audacityteam.org/t/3-4-crash-when-loading-opus-file/88991

**Comment by Leogamr** [comment]
I am having this a lot. So often that I am having to save backups of backups.

![Screenshot (532)](https://github.com/audacity/audacity/assets/151084569/16c209ea-b268-420a-ac95-7523a0e124f4)

**Comment by dozzzzer** [comment]
@Leogamr as for the "Audacity Configuration Error" you're having, it may be happening due to the Read-Only permissions of the `audacity.cfg` file in `C:\Users\<user>\AppData\Roaming\audacity`. Please make sure if the "Read-only" flag is unchecked in the file's Properties dialog.

**Comment by Fydon** [comment]
When Audacity is failing to send crash reports, what is the best way to send these? I have a repeatable crash when exporting audio

**Comment by LWinterberg** [comment]
@Fydon if you have a repeatable crash, please create a new bug with steps to reproduce

**Comment by LWinterberg** [comment]
I just noticed, the generation of the crash report appears to get scrambled.  In the report in #6733, the last couple of lines are

```
0x7ffbe5b30000 - 0x7ffbe5d27fff  ntdll.dll  10.0.19041.4522  (WARNING: No symbols, ntdll.pdb, 4C328D42D0B9A2B218FB85860DECF27C1)
e3310000 - 0x7ffbe335dfff  cfgmgr32.dll  10.0.19041.3996
0x7ffbe3360000 - 0x7ffbe33fcfff  msvcp_win.dll  10.0.19041.3636
0x7ffbe3520000 - 0x7ffbe3546fff  bcrypt.dll  10.0.19041.3636
   [skipped 21 lines in this paste] 
0x7ffbe5840000 - 0x7ffbe58dffff  sechost.dll  10.0.19041.4522
0x7ffbe59a0000 - 0x7ffbe59cefff  IMM32.DLL  10.0.19041.4474
0x7ffbe59d0000 - 0x7ffb
```

Note the truncated front of the second line in the paste (source line 2259) and the truncated end of the last line.

In the report in https://github.com/audacity/audacity/issues/5858, it ends with: 

```
0x76e00000 - 0x76f1ffff  ucrtbase.dll  10.0.19041.3636  (WARNING: No symbols, ucrtbase.pdb, 8C37496DA268C5EC372991A1887D869F1)
0x76f30000 - 0x770d3fff  ntdll.dll  10.0.19041.3693  (WARNING: No symbols, wntdll.pdb, AE4A9CEFBA27BD00DEBC6271B9E3E4BE1)
x75040000 - 0x750b7fff  sechost.dll  10.0.19041.3930
0x750c0000 - 0x7533ffff  combase.dll  10.0
```

Note the truncated front of the second-to-last line and the truncated end of the last line.

**Comment by vsverchinsky** [comment]
> I just noticed, the generation of the crash report appears to get scrambled

That could be UI/print issue. The crash report itself has minidump format, which is binary.

**Comment by DavidBailes** [comment]
Given the number of reports of 3.6.x crashing, wouldn't it be very helpful if this was fixed?

**Comment by scramblr** [comment]
I am also having this issue, and indeed I've noticed a much higher incidence of crashes lately.

**Comment by Dan33185** [comment]
Having this issue as well today...how should I go about reporting crashes/bugs if this doesn't work?

**Comment by cmicek1** [comment]
Ditto today after updating to 3.7.0.

**Comment by Moonwards** [comment]
Also having this problem in Windows 11 using 3.7.3, which I just upgraded to. I continue to not be able to send a crash report. 

I guess I'll open a new issue and paste in the reports.

**Comment by kvanderhaeghen-aleacsysonline** [comment]
3.7.7 Still has this issue.
[audacity-crash-log.txt](https://github.com/user-attachments/files/25262988/audacity-crash-log.txt)

<img width="631" height="633" alt="Image" src="https://github.com/user-attachments/assets/62b5dd2a-7d3e-4467-ae3a-f5723eef543e" />

**Comment by AldebaraanMKII** [comment]
3.7.8 still has this issue.
